features/basics/functype.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Fetching contracted function type
+
+  You can use `functype(name)` method for that:
+
+  ```ruby
+  functype(:add) # => "add :: Num, Num => Num"
```

Comment:
Is that really something "normal" end users are going to use? Otherwise I would move that to "features/developers/..." or something similar.